### ISSUE TITLE
Saved replies?

### ISSUE THREAD

**Issue by noamross** [ropensci/software-review-meta#89]
On a commcall today @Bisaloo noted "Saved Replies" for GitHub Issues: https://docs.github.com/en/github/writing-on-github/working-with-saved-replies/using-saved-replies

What might these be useful for here?
 - Boilerplate for scope-based decisions?
 - Hard-to-remember / longer bot commands?

@Bisaloo, do you have an example of these replies in use (with bots)?

**Comment by Bisaloo** [comment]
I have just saved many [whedon commands](https://joss.readthedocs.io/en/latest/whedon.html) because I always get confused otherwise. For example, I have:

`@whedon add @username as reviewer` (because I mix it with `@whedon assign @username as reviewer`)

`@whedon generate pdf from branch custom-branch-name` (because I don't remember if I need to say "from branch XXX" or directly "from XXXX")

They don't really qualify as "hard-to-remember" or "long" commands :grimacing::sweat_smile:  but I have no interest in keeping these commands in memory and it's annoying to have to check the documentation each time. So these saved replies are really useful for me.

Beyond bot commands, I have also saved some [sample messages](https://joss.readthedocs.io/en/latest/editing.html#sample-messages-for-authors-and-reviewers). I cannot remember if this was an advice from Arfon or not.
